salt/journal-cms/init.sls

```diff
@@ -98,6 +98,16 @@ site-settings:
             #- composer-drupal-scaffold
         - require_in:
             - site-install
+
+api-dummy-nginx-vhost-dev:
```

Comment:
this state definition is already present in `salt/journal-cms/api-dummy.sls`, so it should be removed from this file